app/core/logger.ts

```diff
@@ -55,13 +55,23 @@ export class Logger {
 
         global && this.global();
 
-        if ( store ) this.store();
+        if ( store || this._loadLevel() ) this.store();
 
     }
 
-    private _loadLevel = (): Level => Number(localStorage.getItem( this._storeAs ));
-
-    private _storeLevel(level: Level) { localStorage[ this._storeAs ] = level; }
+    private _loadLevel(): Level {
+        if(typeof localStorage === 'undefined'){
+            return Level.OFF;
+        }else{
+            return Number(localStorage.getItem( this._storeAs ));
+        }
+    };
```

Comment:
In fact this whole method could be written in a single line with something like:
`return localStorage ? Number(localStorage.getItem( this._storeAs )) :  undefined;`